assets/Product.wxs

```diff
@@ -84,17 +84,17 @@
         </DirectorySearch>
       </DirectorySearch>
     </Property>
-    <Condition Message="$(env.ProductName) requires the Universal C Runtime to be installed. You can find a download link to it here: https://github.com/PowerShell/PowerShell/blob/master/docs/installation/windows.md#prerequisites">
+    <Condition Message="$(env.ProductName) requires the Universal C Runtime to be installed to enable remoting over WinRM. You can find a download link to it here: http://aka.ms/pscore6-prereq">
```

Comment:
https?

Reply:
@daxian-dbw You also need to adapt the link in the installer test [here](https://github.com/daxian-dbw/PowerShell/blob/fixmsi/test/powershell/Installer/WindowsInstaller.Tests.ps1#L17)

Reply:
Thanks, will fix.